### DISCUSSION TITLE
Generate Add Data Script

### DISCUSSION THREAD

**Discussion by AndrewMC6** [udotdevelopment/ATSPM#210] [Q&A] [answered]
Hello,

Working through a new installation, however it appears that the GenerateAddDataScript folder in section 2.6 of the installation manual, has a few sub-folders but they are all empty? Any insight would be helpful. Thanks!

**Comment by AndrewMC6** [comment]
It's also worth noting that almost all of the scripts to set in the task scheduler don't have the 'release' folder. There are typically only debug folders present

**Comment by avetraffic** [comment, marked as answer]
Thanks for bringing this up. We've fixed both these issues now as mentioned in #212.
